<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionForm</class>
 <widget class="QWidget" name="OptionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MGAWidgetExtension" name="firstWidget" native="true">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <widget class="QPushButton" name="firstButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>90</width>
         <height>70</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MGAWidgetExtension" name="secondWidget" native="true">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <widget class="QPushButton" name="secondButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>90</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MGAWidgetExtension" name="thirdWidget" native="true">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <widget class="QPushButton" name="thirdButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>90</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>481</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QGroupBox" name="StylegroupBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>351</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QComboBox" name="styleComboBox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>161</width>
          <height>25</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </widget>
      <widget class="QDialogButtonBox" name="changingButtonBox">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>10</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 57 italic 9pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="ServerConfiggroupBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>351</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="IPLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Server IP</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>16</width>
     <height>421</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MGAWidgetExtension</class>
   <extends>QWidget</extends>
   <header>mgawidgetextension.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
